redis-store.gemspec

```diff
@@ -28,4 +28,6 @@ Gem::Specification.new do |s|
   s.add_development_dependency 'redis-store-testing'
   s.add_development_dependency 'appraisal', '~> 2.0'
   s.add_development_dependency 'rubocop', '~> 0.54'
+
+  s.metadata['changelog_uri'] = 'https://github.com/redis-store/redis-store/releases'
```

Comment:
Would be great to have `bug_tracker_uri` & `source_code_uri` too
e.g.
https://github.com/AssetSync/asset_sync/blob/5488d45b619eb4be714f96d3eaa7b8ab53126a03/asset_sync.gemspec#L43-L47